test-case/manual/check-simple-probe-capture.md

```diff
@@ -0,0 +1,77 @@
+# Probes test
+Simple probes functionality test.
+
+# Preconditions
+1. Linux kernel should have these config options enabled:
+   CONFIG_SND_SOC_SOF_DEBUG_PROBES=y
+   CONFIG_SND_SOC_SOF_HDA_PROBES=y
```

Comment:
I would note that these options are _not_ user selectable.

Reply:
Should be now fixed.